blt/src/Blt/Plugin/Commands/CircleCiCommands.php

```diff
@@ -273,7 +273,7 @@ protected function runBehatTest(array $sites) {
   protected function setupSite() {
     $tasks[] = $this->installDependencies();
     $tasks[] = $this->waitForDatabase();
-    $tasks[] = $this->taskExec('service apache2 start');
+    $tasks[] = $this->taskExec('apachectl stop; apachectl start');
```

Comment:
restart or reload not work?

Reply:
It does, but stop and restarting allows for `$_ENV` variables to pass into apache.